blueocean-core-js/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@jenkins-cd/blueocean-core-js",
-  "version": "0.0.45",
+  "version": "0.0.46-unpublished",
```

Comment:
why this? 👎  agree with @tfennelly we should just not keep adding random suffixes to the version